dist/ci/travis_script.sh

```diff
@@ -46,15 +39,16 @@ if test -z "$SUBTEST"; then
       pushd ../backend
       make test_unit
       ;;
+    migrate)
+      mv db/structure.sql db/structure.sql.git
+      bundle exec rails db:drop db:create || exit 1
+      xzcat test/dump_2.5.sql.xz | sed '/^USE api_.*;/d' | mysql ci_api_test
```

Comment:
because it is not the same scenario as with creating the 2.5 database with the rails stack from 2.5. And it shows in the different structure.sql actually.